build/Targets/RepoToolset/Imports.targets

```diff
@@ -35,4 +35,23 @@
   <Import Project="OptimizationData.targets"/>
   <Import Project="SymStore.targets"/>
 
+  <UsingTask TaskName="Microsoft.DotNet.Arcade.Sdk.LocateDotNet" AssemblyFile="$(RoslynToolsBuildTasksAssembly)" />
+
+  <Target Name="InitializeDotNetToolPath" Condition="'$(DotNetTool)' == ''">
```

Comment:
Does this mean that VS based builds could pick up the dotnet install that our scripts bootstrap?

Reply:
So is that a yes? Or do we still have to run the .NET SDK MSI before we can build Roslyn?

Reply:
The target will fail if it can't find the right version of .NET SDK. It won't try to install it.